holoviews/core/dimension.py

```diff
@@ -3,6 +3,8 @@
 axis or map dimension. Also supplies the Dimensioned abstract
 baseclass for classes that accept Dimension values.
 """
+from __future__ import unicode_literals
+
```

Comment:
Probably not a bad idea to add some of the ``__future__`` imports everywhere including ``division``, ``print_function`` and this one.